frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -55,21 +39,20 @@ export function PersonModal({ visible, view, filters, onSaveCohort }: Props): JS
     const title = useMemo(
         () =>
             isInitialLoad ? (
-                'Loading persons list...'
+                'Loading users…'
```

Comment:
Aren't we calling this "persons" for consistency everywhere?

Reply:
Wellll, I specifically renamed this because in the graph hover tooltip it's "inspect users". So I do agree the terminology between "users", "persons", and "people" is kind of wavering.

Reply:
Okay, updated, but in a PR on top of this one: https://github.com/PostHog/posthog/pull/5739

Reply:
Agree that it should be persons. We've been inconsistent with this in the past, but looking forward we're going with "persons".